petprep/workflows/pet/outputs.py

```diff
@@ -139,13 +83,7 @@ def prepare_timing_parameters(metadata: dict):
     if len(slice_timing) > 1:
         st = sorted(slice_timing)
         TA = st[-1] + (st[1] - st[0])  # Final slice onset + slice duration
-        # For constant TR paradigms, use DelayTime
-        if 'RepetitionTime' in timing_parameters:
-            TR = timing_parameters['RepetitionTime']
-            if not np.isclose(TR, TA) and TA < TR:
-                timing_parameters['DelayTime'] = TR - TA
-        # For variable TR paradigms, use AcquisitionDuration
-        elif 'VolumeTiming' in timing_parameters:
+        if 'VolumeTiming' in timing_parameters:
             timing_parameters['AcquisitionDuration'] = TA
```

Comment:
I can't imagine you need this function at all, since there are no gaps in PET recording.

Reply:
Thanks for the feedback, Chris - in general, still need to remove a bunch of slice timing related stuff.